.github/workflows/build-images.yml

```diff
@@ -0,0 +1,77 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
```

Comment:
Personally I like to build on all branches and just not push the result to the registry then. It ensures the container still builds. But that's probably a decision for the maintainers.

Reply:
I am concerned by our energy footprint, so I try to limit the automated builds to a minimum.
When working on making changes to one of the Docker images, they need to be tested locally anyway, so au auto-build does not add a lot of value. And if the builds fail once in a while, it is not a big deal as it does not affect production and can be fixed or reverted with another PR.